### ISSUE TITLE
check_subrc not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1827]
```abap
DATA bar TYPE i VALUE 1.
SELECT *
  INTO TABLE @DATA(lt_tab)
  FROM voided
  WHERE foo = @bar
  ORDER BY PRIMARY KEY.
```
